.azure-pipelines/all.yml

```diff
@@ -27,11 +27,11 @@ resources:
       DD_HOSTNAME: none
       DD_INSIDE_CI: 'true'
 variables:
-  - group: extras-variables
-  - name: DDEV_COLOR
-    value: 1
-  - name: DD_TRACE_AGENT_PORT
-    value: 8127
+- group: extras-variables
+- name: DDEV_COLOR
+  value: 1
+- name: DD_TRACE_AGENT_PORT
+  value: 8127
```

Comment:
Please leave this part untouched as it's unrelated to the integration you're adding (my guess is probably your editor is autoformatting this).

Reply:
this seems to be changed by one of the `ddev validate` commands...  
my latest changes reverted this as shown, yet after running the validate commands it was back.  
possibly a parsing issue??

here are the validation commands I ran for your ref:  

```
ddev test filemage
ddev validate ci --fix && ddev -x validate config -s filemage && ddev validate models filemage -s && ddev validate all filemage
ddev validate all filemage
ddev validate codeowners
```

I am not 100% sure I am understanding your CI pipeline but I would wager a guess that `ddev validate ci --fix` may be changing this section.

Reply:
resolved in 992251647c28e871d4e0357945a0a4f169d8293d